test/jwa.test.js

```diff
@@ -73,15 +74,15 @@ BIT_DEPTHS.forEach(function (bits) {
   });
 });
 
-test('ECDSA signing, verifying', function (t) {
-  const input = 'kristen schaal';
-  BIT_DEPTHS.forEach(function (bits) {
+BIT_DEPTHS.forEach(function (bits) {
```

Comment:
This made it easier for me while debugging (titles in `make verbose`).

Can switch this back if you like (didn't realize I left it in).